Index.html

```diff
@@ -32,8 +32,53 @@ <h2>I’m a software developer</h2>
          <li><a><img src="media/github.svg" alt="github link"></a></li>
          <li><a><img src="media/angellist.svg" alt="angellint link"></a></li>
         </ul>
+
+        <a class="disable-button"><img src="media/Disabled.svg" alt=""></a>
  
       </div>
+      <section class="work-container">
+        <div class="my-work-text">
+          <h2>My Recent Works</h2>
+          <hr>
+       </div>
+        <div class="work-history">
+          <a><img src="media/Enable.svg" alt="Portfolio"></a>
+          <a><img src="media/Enable.svg" alt="Portfolio"></a>
+          <a><img src="media/Enable.svg" alt="Portfolio"></a>
+          <a><img src="media/Enable.svg" alt="Portfolio"></a>
+          <a><img src="media/Enable.svg" alt="Portfolio"></a>
+          <a><img src="media/Enable.svg" alt="Portfolio"></a>
+          <a><img src="media/Enable.svg" alt="Portfolio"></a>
+        </div>
+      </section>
+      <section class="about-me">
+        <div class="about-me-text">
+          <h2>About me</h2>
+          <p> I can help you build a product , feature or website Look through some of
+             my work and experience! If you like what you see and have a project you need coded, 
+            don’t hesitate to contact me.
+          </p>
```

Comment:
- [ ] Nice job implementing the About myself section :+1: However there is too much space between the `About myself` section and the works (see images). Kindly reduce it, in order to follow the design.

This is what your website looks like:
![Screenshot from 2023-03-25 01-33-57](https://user-images.githubusercontent.com/91142432/227681722-23f269ca-2ea4-4df0-9c0e-ed1a323eb6d7.png).

And how it suppose to look like:
![Screenshot from 2023-03-25 02-23-24](https://user-images.githubusercontent.com/91142432/227681864-824c39c6-9407-4d7f-b532-871e799e8855.png)

Reply:
resolved